docs/StepbyStepManualDeploymentGuideScalarDLOnAzure.md

```diff
@@ -0,0 +1,237 @@
+# Example steps to Deploy Scalar DL on Azure
+
+This guide assumes that you create the environment on the basis of [manual deployment guide for Azure](./ManualDeploymentGuideScalarDLOnAzure.md).
+This document contains example steps for creating a Scalar DL environment in the Azure cloud.
+
+## Prerequisites
+
+You can skip the following steps If the user has an [Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner) or [Contributor](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#contributor) role in Azure.
```

Comment:
`Contributor` doesn't seem to have role assignment privileges in` rbac`, is that wrong?
```
Grants full access to manage all resources, but does not allow you to assign roles in Azure RBAC, manage assignments in Azure Blueprints, or share image galleries.
```

It seems that the `Owner` or `User Access Administrator` can assign roles.
https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=current#prerequisites

Reply:
Thank you, You are correct only `owner` and `User Access Administrator` can assign roles. 
Updated in aec4f67